dev/com.ibm.ws.jaxrs.2.0.common/test/com/ibm/ws/jaxrs/utils/test/URIParseTest.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxrs.utils.test;
+
+import java.net.URI;
+
+import org.apache.cxf.jaxrs.utils.HttpUtils;
+import org.junit.Test;
+
+import junit.framework.Assert;
+
+/**
+ *
+ */
+public class URIParseTest {
+
+
+    @Test
+    public void testURIs() throws Exception {
```

Comment:
It would be good to have some invalid URIs in this test as well to see the behavior is Exception for new URI and null for the new method.  Maybe just a new testInvalidURIs test or something like that.